gammapy/cube/make.py

```diff
@@ -395,21 +405,59 @@ def make_mask_energy_edisp_bias(self, dataset):
 
         Parameters
         ----------
-        dataset : `SpectrumDataset` or `SpectrumDatasetOnOff`
+        dataset : `MapDataset`, `MapDatasetOnOff`, `SpectrumDataset` or `SpectrumDatasetOnOff`
             Dataset to compute mask for.
 
         Returns
         -------
         mask_safe : `~numpy.ndarray`
             Safe data range mask.
         """
+        edisp = dataset.edisp
+
         if isinstance(dataset, (MapDataset, MapDatasetOnOff)):
-            raise NotImplementedError(
-                "'edisp-bias' method currently only supported for spectral datasets"
-            )
+            if self.position is None:
+                raise AttributeError(
+                    "To compute the energy bias, a position has to be set"
+                )
+            e_reco = dataset.counts.geom.get_axis_by_name("energy").edges
+            edisp = edisp.get_energy_dispersion(self.position, e_reco)
+            counts = dataset.counts.geom
+        else:
+            counts = dataset.counts
 
-        e_min = dataset.edisp.get_bias_energy(self.bias_percent / 100)
-        return dataset.counts.energy_mask(emin=e_min)
+        e_min = edisp.get_bias_energy(self.bias_percent / 100)
+        return counts.energy_mask(emin=e_min)
+
+    def make_mask_bkg_peak_energy(self, dataset):
+        """Make safe energy mask based on the binned background.
+
+        The energy threshold is defined as the upper edge of the energy
+        bin with the highest predicted background rate. This method is motivated
+        by its use in the HESS DL3 validation paper: https://arxiv.org/pdf/1910.08088.pdf
+
+        Parameters
+        ----------
+        dataset : `MapDataset`, `MapDatasetOnOff`, `SpectrumDataset` or `SpectrumDatasetOnOff`
+            Dataset to compute mask for.
+
+        Returns
+        -------
+        mask_safe : `~numpy.ndarray`
+            Safe data range mask.
+        """
+
+        if isinstance(dataset, (MapDataset, MapDatasetOnOff)):
+            background_spectrum = dataset.background_model.map.get_spectrum()
+            counts = dataset.counts.geom
+        else:
+            background_spectrum = dataset.background
+            counts = dataset.counts
+
+        peak = background_spectrum.data.max()
+        idx = list(background_spectrum.data).index(peak)
+        e_min = background_spectrum.energy.center[idx]
```

Comment:
Note the implementation differs from the description "upper edge of the energy bin with the highest predicted background rate". Please unify to whatever is described in the paper...